frontend/src/app/_components/creator-page/creator-page.component.ts

```diff
@@ -12,6 +12,7 @@ export class CreatorPageComponent implements OnInit {
   constructor() { }
 
   ngOnInit(): void {
+    // TODO check for undefined creator & handle. 404?
```

Comment:
The way to handle it would be a console message or alert that only shows up in devel builds. Not sure how possible it would be.